T_SPQ_001 - AssertionError: 'TEST-STOCK-ITEM' not found in []

### Information about bug

======================================================================
FAIL: test_execute_with_item_code_filter_T_SPQ_001 (erpnext.stock.report.stock_projected_qty.test_stock_projected_qty.TestStockReorderReportExecute)
Ensure only filtered item_code is returned in execute()
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/ubuntu/frappe-bench/apps/erpnext/erpnext/stock/report/stock_projected_qty/test_stock_projected_qty.py", line 39, in test_execute_with_item_code_filter_T_SPQ_001
    self.assertIn(self.item_code.name, item_codes)
AssertionError: 'TEST-STOCK-ITEM' not found in []

----------------------------------------------------------------------
Ran 1 test in 1.150s

FAILED (failures=1)

### Module

stock

### Version

Assets: v0.0.1 (pre-prod)

ERPNext: v15.58.2 (pre-prod)

ERPNext CRM: v0.0.1 (pre-prod)

ERPNext India: v0.0.1 (pre-prod)

Frappe Framework: v15.63.1 (pre-prod)

India Compliance: v15.17.1 (pre-prod)

Payments: v0.0.1 (version-15)

Projects: v0.0.1 (pre-prod)

Sales Commission: v0.0.1 (pre-prod)

### Installation method

None

### Relevant log output / Stack trace / Full Error Message.

```shell

```

--- a/erpnext/stock/report/stock_projected_qty/test_stock_projected_qty.py
+++ b/erpnext/stock/report/stock_projected_qty/test_stock_projected_qty.py
@@ -32,6 +32,16 @@ class TestStockReorderReportExecute(FrappeTestCase):
 		if not frappe.db.exists("UOM", self.uom_name):
 			frappe.get_doc({"doctype": "UOM", "uom_name": self.uom_name}).insert()
 
+		# Create stock entry for projected_qty
+		frappe.get_doc(
+			{
+				"doctype": "Stock Entry",
+				"stock_entry_type": "Material Receipt",
+				"company": self.company,
+				"items": [{"item_code": self.item_code.name, "qty": 5, "t_warehouse": self.warehouse}],
+			}
+		).insert().submit()
+
 	def test_execute_with_item_code_filter_T_SPQ_001(self):
 		"""Ensure only filtered item_code is returned in execute()"""
 		columns, data = execute({"item_code": "TEST-STOCK-ITEM"})

